infra.bs

```diff
@@ -84,6 +85,73 @@ either width or height is zero"), means an inclusive "or" (implying "or both"),
 out as being exclusive (with "but not both").
 
 
+<h3 id=privacy>Privacy concerns</h3>
+
+<p>Some features that are defined in documents using the Infra Standard might trade user convenience
+for a measure of user privacy.
+
+<p>In general, due to the Internet's architecture, a user can be distinguished from another by the
+user's IP address. IP addresses do not perfectly match to a user; as a user moves from device to
+device, or from network to network, their IP address will change; similarly, NAT routing, proxy
+servers, and shared computers enable packets that appear to all come from a single IP address to
+actually map to multiple users. Technologies such as onion routing can be used to further
+anonymize requests so that requests from a single user at one node on the Internet appear to come
+from many disparate parts of the network.
+
+<p>However, the IP address used for a user's requests is not the only mechanism by which a user's
+requests could be related to each other. Cookies, for example, are designed specifically to enable
+this, and are the basis of most of the Web's session features that enable you to log into a site
+with which you have an account.
+
+<p>There are other mechanisms that are more subtle. Certain characteristics of a user's system can
+be used to distinguish groups of users from each other; by collecting enough such information, an
+individual user's browser's "digital fingerprint" can be computed, which can be as good, if not
+better, as an IP address in ascertaining which requests are from the same user.
```

Comment:
This acknowledges fingerprinting but not "covert state" like HSTS supercookies or in general abuse of stateful mechanisms that are not meant to hold an identifier. (Technically, explicit state intended for storage is also a tracking vector by many  definitions of tracking).

Reply:
I've tried to cover shared state and caches now.